BASE/src/Microsoft.ApplicationInsights/DataContracts/TraceTelemetry.cs

```diff
@@ -7,13 +7,14 @@
     using Microsoft.ApplicationInsights.Extensibility;
     using Microsoft.ApplicationInsights.Extensibility.Implementation;
     using Microsoft.ApplicationInsights.Extensibility.Implementation.External;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation.Metrics;
 
     /// <summary>
     /// Telemetry type used for log messages.
     /// Contains a time and message and optionally some additional metadata.
     /// <a href="https://go.microsoft.com/fwlink/?linkid=525722#tracktrace">Learn more</a>
     /// </summary>
-    public sealed class TraceTelemetry : ITelemetry, ISupportProperties, ISupportAdvancedSampling, IAiSerializableTelemetry
+    public sealed class TraceTelemetry : OperationTelemetry, ITelemetry, ISupportProperties, ISupportAdvancedSampling, IAiSerializableTelemetry
```

Comment:
trace should not inherit operation telemetry

Reply:
Fixed